pkgs/applications/misc/actiona/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, wrapQtAppsHook
+
+, bluez
+, libnotify
+, libXdmcp
+, libXtst
+, opencv
+, qtbase
+, qtmultimedia
+, qtscript
+, qttools
+, qtx11extras
+, qtxmlpatterns
+
+, textToSpeechSupport ? false
+, qtspeech
+}:
+
+stdenv.mkDerivation {
+  pname = "actiona";
+  version = "unstable-2022-05-28";
+
+  src = fetchFromGitHub {
+    owner = "Jmgr";
+    repo = "actiona";
+    rev = "3f5a76424da1f8bc6b6225b6b8ac1e3d0c8c9c3d";
+    hash = "sha256-+UH7sbesmDoBG+t85kgwtVm7y2rfMgDj/m8GaSWokvk=";
+    fetchSubmodules = true; # mINI: single header-file helper for .ini file management
```

Comment:
??

Reply:
I guess this comment isn't really needed. It was put there just to signal what the only submodule used way (it was mINI)